data/brands/amenity/restaurant.json

```
@@ -1155,18 +1155,18 @@
       }
     },
     {
-      "displayName": "Chuck E. Cheese's",
+      "displayName": "Chuck E. Cheese",
       "id": "chuckecheeses-85c92c",
-      "locationSet": {"include": ["ca", "us"]},
+      "locationSet": {"include": ["ca", "pr", "us"]},
```

According to their Wikipedia article, they have some locations outside of the US/Canada (Puerto Rico is included in the us country-coder polygon so it isn't necessary to include it): https://en.wikipedia.org/wiki/Chuck_E._Cheese#International_markets
```suggestion
      "locationSet": {"include": ["ae", "bh", "ca", "ch", "co", "gt", "hn", "mx", "pa", "pe", "qa", "sa", "sr", "sv", "tt", "us"]},
```